src/components/Results/DetectedIssueResources.tsx

```diff
@@ -1,17 +1,41 @@
 import React from 'react';
-import { Accordion, AccordionDetails, Grid, Link, Typography, withStyles } from '@material-ui/core';
+import {
+  Accordion,
+  AccordionDetails,
+  Grid,
+  Link,
+  Typography,
+  withStyles,
+  Popover,
+  makeStyles,
+  createStyles,
+  Theme
+} from '@material-ui/core';
 import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
 import { R4 } from '@ahryman40k/ts-fhir-types';
 import { Enums } from 'fqm-execution';
-import { measureFileState } from '../../state';
+import { measureFileState, patientFileState } from '../../state';
 import { useRecoilValue } from 'recoil';
-import { findValueSetInBundle } from '../Helpers';
+import { findResourceInBundle, findValueSetInBundle } from '../Helpers';
 import fhirpath from 'fhirpath';
 
 interface Props {
   detectedIssue: R4.IDetectedIssue;
 }
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    popover: { pointerEvents: 'none' },
+    paper: {
+      padding: theme.spacing(1)
+    },
+    reasonDetail: {
+      paddingLeft: '32px',
+      paddingTop: '12px'
+    }
```

Comment:
```suggestion
    reasonDetail: {
      paddingLeft: '32px',
      paddingTop: '12px',

      '&:hover': {
        cursor: 'pointer'
      }
    }
```

Suggest updating the style so that hovering over the text changes the cursor